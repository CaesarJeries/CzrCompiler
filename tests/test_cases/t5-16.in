void foo(int x) {
    x = x * x;
    printi(x);
    print("\n");
}

void main()
{
    foo(2);
    foo(3);
    foo(10);
}