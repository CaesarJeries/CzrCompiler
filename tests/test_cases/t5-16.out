Loaded: ./exceptions.s
4
9
100
